rafthttp/stream.go

```diff
@@ -287,7 +307,7 @@ func (cr *streamReader) decodeLoop(rc io.ReadCloser) error {
 	case streamTypeMessage:
 		dec = &messageDecoder{r: rc}
 	default:
-		log.Panicf("rafthttp: unhandled stream type %s", cr.t)
+		log.Panicf("rafthttp: unhandled stream type %s", t)
```

Comment:
we should never panic like this. this is triggered by an external input... but we can change this later.